### ISSUE TITLE
Something is mangling input file names

### ISSUE THREAD

**Issue by cdlm** [pillar-markup/pillar#66]
The dash disappears between the actual `$PWD` and what Pillar seems to be using as `baseDirectory`:
```
$ echo $PWD
/Users/damien/Documents/Pharo/lan-simulator/exo
$ ./pillar export
==== Startup Error: CCConfigurationError: /Users/damien/Documents/Pharo/lansimulator/exo is not a valid directory.
```

**Comment by DamienCassou** [comment]
@Ducasse can reproduce the problem. I'm having a look now

**Comment by DamienCassou** [comment]
```
Name: Pillar-Cli-DamienCassou.17
Author: DamienCassou
Time: 7 September 2015, 3:46:19.696843 pm
UUID: c89989a3-40d6-436c-914d-b1c3e0c81aa0
Ancestors: Pillar-Cli-CyrilFerlicot.16

Fix for issue #66 "Something is mangling input file names".

Pillar was eating '-' and '=' characters while reading command-line parameters.
```
